libunwind/src/Unwind-wasm.c

```diff
@@ -35,7 +32,12 @@ struct _Unwind_LandingPadContext {
 
 // Communication channel between compiler-generated user code and personality
 // function
-thread_local struct _Unwind_LandingPadContext __wasm_lpad_context;
+#if __STDC_VERSION__ >= 202311L
```

Comment:
We always compile the built library with C++23, so I don't understand this change.

Reply:
thread.h does not exist.

Reply:
> ldionne

it is a c file. not a .cpp file. it compiles with C compiler not C++